awx/api/urls/urls.py

```diff
@@ -136,6 +140,8 @@
     re_path(r'^activity_stream/', include(activity_stream_urls)),
     re_path(r'^workflow_approval_templates/', include(workflow_approval_template_urls)),
     re_path(r'^workflow_approvals/', include(workflow_approval_urls)),
+    re_path(r'^bulk/host_create/$', BulkHostCreateView.as_view(), name='bulk_host_create'),
```

Comment:
I think it would be a good idea to break out all of the bulk urls into their own separate urls module, like many of the ones above.